README.md

```diff
@@ -6,15 +6,18 @@
 [![Documentation](https://docs.rs/rand/badge.svg)](https://docs.rs/rand)
 [![Minimum rustc version](https://img.shields.io/badge/rustc-1.22+-yellow.svg)](https://github.com/rust-lang-nursery/rand#rust-version-requirements)
 
-A Rust library for random number generators and other randomness functionality.
+A Rust library for random number generation.
 
-See also:
+Rand provides utilities to generate random numbers, to convert them to useful
+types and distributions, and some randomness-related algorithms.
 
-*   [rand_core](https://crates.io/crates/rand_core)
+Part of the functionality of Rand lives in the [rand_core](
+https://crates.io/crates/rand_core) crate, which is not intended for users but
+primarily for crates implementing RNGs.
```

Comment:
I don't like this style much. We will probably have more sub-crates shortly so a list is a good presentation, and then a description something like:

> core generation traits; this crate is most useful when implementing RNGs

Reply:
I have changed the text a bit, but shall we wait switching to a list until it contains more than one item?

Reply:
If you like. But you're the one *switching* here.